src/com/owncloud/android/lib/common/operations/RemoteOperationResult.java

```diff
@@ -266,7 +267,27 @@ public RemoteOperationResult(boolean success, HttpMethod httpMethod) throws IOEx
             }
         }
 
-        if (mHttpCode == HttpStatus.SC_SERVICE_UNAVAILABLE) {
+        if (mHttpCode == HttpStatus.SC_UNSUPPORTED_MEDIA_TYPE) {    // 415
+
+            String bodyResponse = httpMethod.getResponseBodyAsString();
```

Comment:
Time to refactor this block. The code in this 'if' is repeated several time. Let's define a private method to reduce duplication.